ui/components/Navbar/Navbar.styles.js

```diff
@@ -314,6 +324,21 @@ export const useStyles = makeStyles((theme) => ({
   rightTranslate: {
     transform: "translateX(0.5px)",
   },
+  hideScrollbar: {
```

Comment:
Hiding scrollbar is a common utility that we want to be able to reuse in our project. Let us make this generic.

Reply:
Yes

Reply:
@sudo-NithishKarthik where should i keep the css component??

Reply:
Ideally, you would want to create a HOC that we can use to wrap any component whose scrollbar is supposed to be hidden.

Reply:
You would create a `Component A` which would accept another component called `Component B` in it's prop and `<ComponentA comp={ComponentB} />` would render `ComponentB` without the scrollbar or the custom scrollbar styles applied.